cargo-bpf/src/llvm.rs

```diff
@@ -0,0 +1,121 @@
+use std::path::Path;
+use llvm_sys::analysis::{LLVMVerifierFailureAction::*, LLVMVerifyModule};
+use llvm_sys::core::*;
+use llvm_sys::debuginfo::LLVMStripModuleDebugInfo;
+use llvm_sys::initialization::*;
+use llvm_sys::ir_reader::LLVMParseIRInContext;
+use llvm_sys::prelude::*;
+use llvm_sys::target::*;
+use llvm_sys::{LLVMAttributeFunctionIndex, LLVMAttributeReturnIndex, LLVMInlineAsmDialect::*};
+use std::ffi::{CStr, CString};
+use std::os::raw::c_char;
+use std::ptr;
+
+pub fn process_ir(input: &Path, output: &Path) -> Result<(), String> {
```

Comment:
It might make sense to break up this function to logical parts, or at least put in comments. It's not in a performance critical path, so I'm sure we could get away with simply splitting it up

Reply:
Done